packages/flutter_tools/templates/app_shared/android.tmpl/settings.gradle.tmpl

```diff
@@ -10,11 +10,19 @@ pluginManagement {
 
     includeBuild("${settings.ext.flutterSdkPath}/packages/flutter_tools/gradle")
 
+    repositories {
+        gradlePluginPortal()
+        google()
+    }
```

Comment:
Re https://github.com/flutter/flutter/pull/127897#issuecomment-1715906594

Long shot: is it possible the tests are running in parallel in isolated environments, but from the same-ish IP range, so the shards are DDoS'ing gradlePluginPortal/JCenter and get rate limited with a strange error message? Or simply JCenter is flaky enough to fail with a high enough percentage that there's always a failure?

Either way, @you can probably prevent failures if you [put](https://docs.gradle.org/current/userguide/plugins.html#sec:custom_plugin_repositories) `mavenCentral()` in front of `gradlePluginPortal` to ensure that less artifacts are hitting JCenter. The artifact in question is available in MavenCentral too: https://mvnrepository.com/artifact/io.netty/netty-transport/4.1.52.Final

The weird thing is: why this PR? The default is `repositories { gradlePluginPortal() }` so this didn't change much... although, the 2 new plugins below might result in more files being resolved?

Reply:
Sure, after CI passes for [`4109747`](https://github.com/flutter/flutter/pull/127897/commits/410974794f249d2d73fec676a3ebd47a2e69304d) (what a funny commit number btw!) I'm happy to do it.

Reply:
Reverted [`29b01c3`](https://github.com/flutter/flutter/commit/29b01c39ad9124dea6d4c493352fe379a9ba8c8e) in [`a39fc08`](https://github.com/flutter/flutter/pull/127897/commits/a39fc08fd92d646d1b8f44b8a4933dd872524990) and the same error appeard.

Reply:
Thanks, so that confirms to me that gPP/jcenter can't handle missing artifacts (at least not a lot of them at once).
In the CI of https://github.com/flutter/flutter/pull/127897/checks?check_run_id=16762079212
this was the error https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8770052323773862369/+/u/run_test.dart_for_tool_integration_tests_shard_and_subshard_1_4/test_stdout#L4443_17
which is a different artifact than the original Netty one.
I might try to reproduce it in some way and report to Gradle if possible.

(re funny commit number: https://github.com/not-an-aardvark/lucky-commit)

Reply:
I tried sending 1000 requests with 200-1000 parallelism for a `.pom`, but didn't get this "peer" error. Only socket closed and connection reset.